### PR TITLE
Update for Gradle

### PR DESCRIPTION
**Pull request by dblitza**
update for gradle

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mattbsox) [inline .classpath]: remove this file

File: .project [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mattbsox) [inline .project]: remove this file

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    repositories {
+        mavenLocal()
+        maven {
+            name = 'Sonatype Nexus Snapshots'
+            url = 'https://oss.sonatype.org/content/repositories/snapshots/'
+        }
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'net.wasdev.wlp.gradle.plugins:liberty-gradle-plugin:1.1-SNAPSHOT'
```
Comment (mattbsox) [inline build.gradle]: update to 2.0-SNAPSHOT

File: build.gradle [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+
+war {
+    archiveName = baseName + '.' + extension
+}
+    
+liberty {
+    server {
+        configFile = file('src/main/liberty/config/server.xml')
```
Comment (mattbsox) [inline build.gradle:68]: Don't need this if it is the default value `src/main/liberty/config/server.xml`

File: build.gradle [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+        bootstrapProperties = ['default.http.port': testServerHttpPort, 
+                               'default.https.port': testServerHttpsPort,
+                               'appContext': warContext,
+                               'appLocation': war.archiveName]
+        looseApplication = true
```
Comment (mattbsox) [inline build.gradle]: LooseApplication is set to true by default. Might want to leave this in for transparency.

File: build.gradle [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+task printMessageAboutRunningServer {
+    doLast {
+        println "The server is now running at http://localhost:${testServerHttpPort}/${warContext}"
+        println "To stop the server run 'gradle libertyStop'"
+    }
+}
+
+clean.dependsOn 'libertyStop'
```
Comment (mattbsox) [inline build.gradle]: remove all of these except for `libertyStart.finalizedBy 'printMessageAboutRunningServer'`

File: build.gradle [hunk 7]
```diff
@@ -0,0 +1,115 @@
+apply plugin: 'maven'
+apply plugin: 'war'
+apply plugin: 'liberty'
+
+group = 'liberty.samples'
+version = '1'
+
+description = "Liberty Spring Gradle example"
+
+sourceCompatibility = 1.7
```
Comment (mattbsox) [inline build.gradle]: Update these 2 compatibility properties to 1.8

**Review by mattbsox — CHANGES_REQUESTED** [review]

**Review by cthigh — APPROVED** [review]

**Review by cthigh — APPROVED** [review]
Looks good

**Pull request merged**